app/actions/views/root.js

```diff
@@ -51,7 +51,7 @@ export function goToNotification(notification) {
             await loadChannelsIfNecessary(teamId)(dispatch, getState);
         } else {
             await selectFirstAvailableTeam()(dispatch, getState);
-            teamId = currentTeamId;
+            teamId = state.entities.team.currentTeamId;
```

Comment:
If `teamId` was falsey, then that means `currentTeamId` was empty. We need to go back to the store now that we've selected the first available team